docker/puppetdb/docker-entrypoint.d/30-configure-ssl.sh

```diff
@@ -10,7 +10,7 @@ if [ ! -f "${SSLDIR}/certs/${CERTNAME}.pem" ] && [ "$USE_PUPPETSERVER" = true ];
   if [ -n "${DNS_ALT_NAMES}" ]; then
     DNS_ALT_NAMES="${HOSTNAME},$(hostname -s),$(hostname -f),${DNS_ALT_NAMES}"
   fi
-  /ssl.sh "$CERTNAME"
+  CERTNAME="$CERTNAME" /ssl.sh
```

Comment:
Might want to tweak how we set `DNS_ALT_NAMES` and make it consistent with puppetdb, pe-puppetdb, pe-console-services, pe-orchestration-services and pe-boltserver

Reply:
Did this